README.md

```diff
@@ -46,6 +46,15 @@ The workflow is based on the [OpenSwathWorkflow](http://openswath.org/en/latest/
 
 See [usage docs](https://nf-co.re/diaproteomics/usage) for all of the available options when running the pipeline.
 
+## Pipeline Summary
```

Comment:
Either add a pipeline summary or remove the NGS related dummy text

Reply:
done